docs/manual/introduction/Browser-support.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<base href="../../" />
+	<script src="list.js"></script>
+	<script src="page.js"></script>
+	<link type="text/css" rel="stylesheet" href="page.css" />
+</head>
+<body>
+	<h1>[name]</h1>
+
+	<h2>Overview</h2>
+	<div>
+		<p>
+			Three.js can use WebGL to render your scenes on all modern browsers. For older browsers, especially Internet Explorer 10 and below, you may have to fallback to one of the other renderers (CSS2DRenderer, CSS3DRenderer, SVGRenderer, CanvasRenderer). Additionally, you may have to include some polyfills, especially if you are using files from the /examples folder.
```

Comment:
Can you add a link to the folder with the other renderers?

>...to one of the other [link:https://github.com/mrdoob/three.js/tree/master/examples/js/renderers renderers]

And also to the /examples folder:

> ...you are using files from the [link:https://github.com/mrdoob/three.js/tree/master/examples /examples] folder